src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1161,38 +1169,60 @@ From there, simply click on a Markdown file or SVG icon in the File Explorer and
   <data name="SettingsWindow_Title" xml:space="preserve">
     <value>PowerToys Settings</value>
   </data>
-  <data name="Run_PositionAppearance_GroupSettings.Text" xml:space="preserve">

```

Comment:
need to verify these removals as well

Reply:
Also not sure why this was removed - I don't think this was the intent. Should I reverse? @crutkas

Reply:
Let's verify the strings exist. My default answer is reverse the delete.

Reply:
Re-adding these.